symbolic-testutils/fixtures/linux/elf_debuglink/generate_elf_debug_link.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# This script was used to generate the fixtures in the "gen" directory, used to test the
+# `ElfObject::debug_link` method.
+
+# Pre-requisites:
+#
+# - a C compiler
+# - eu-elfcompress (elfutils)
+# - objcopy (GNU Binary Utilities)
+# - crc32 (perl-archive-zip)
+
+OUTPUT=gen
+
+# 0. Clean and remake output directory, switch to it
+rm -rf $OUTPUT
+mkdir -p $OUTPUT
+cd $OUTPUT
+
+# 1. compile our C example. To keep size low, let's compile the simplest program we can write.
+cc -x c -Os -o elf_without_debuglink - << EOF
```

Comment:
isn't the debuglink a GCC-specific thing?  should this be directly gcc?  Or is this a POSIX thing?

Reply:
Since the debuginfo is added to the ELF afterwards using `objcopy` and is fake anyway (for testing purposes), I'm not sure if this can pose a problem.

It is no harm to change it though, since apparently I failed to run clippy on the tests and need to update :eyes:.

Reply:
Changed (FTR, I probably missed the clippy error due to running `cargo clippy` instead of `cargo clippy --all-features --workspace --tests --examples -- -D clippy::all` like the CI)